pkg/web/static/styles/style.css

```diff
@@ -75,12 +75,4 @@ body main {
 
 .htmx-request.htmx-indicator{
   display: inline;
-}
-
-/* Set a media query for the transaction list table */
```

Comment:
Attempting to set the dropdown menu's position to absolute didn't resolve the issue, so removing this style is the temporary solution. We may have to use a different component library to display dropdown menus in the tables to ensure that we can resolve the overflow issue and still display the action menu in the last row to users.